osu.Game/Skinning/Editor/SkinEditorOverlay.cs

```diff
@@ -29,13 +26,12 @@ public partial class SkinEditorOverlay : OverlayContainer, IKeyBindingHandler<Gl
 
         protected override bool BlockNonPositionalInput => true;
 
-        [CanBeNull]
-        private SkinEditor skinEditor;
+        private SkinEditor? skinEditor;
 
-        [Resolved(canBeNull: true)]
-        private OsuGame game { get; set; }
+        [Resolved]
+        private OsuGame game { get; set; } = null!;
```

Comment:
Was this dependency on `OsuGame` intentionally changed from optional to required here?

Reply:
Yeah, there were already usages where it wasn't null-checked (see `Update()`) and it does seem required for this overlay to work as expected.

Of note, this is `SkinEditorOvleray`, the component which creates a `SkinEditor` for the current screen.